core/src/test/java/com/linecorp/armeria/server/streaming/JsonLinesTest.java

```diff
@@ -0,0 +1,101 @@
+package com.linecorp.armeria.server.streaming;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.IOException;
+import java.util.stream.Stream;
+
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.google.common.collect.ImmutableList;
+import com.google.common.util.concurrent.MoreExecutors;
+
+import com.linecorp.armeria.client.WebClient;
+import com.linecorp.armeria.common.AggregatedHttpResponse;
+import com.linecorp.armeria.common.HttpData;
+import com.linecorp.armeria.common.HttpObject;
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.HttpStatus;
+import com.linecorp.armeria.common.MediaType;
+import com.linecorp.armeria.common.ResponseHeaders;
+import com.linecorp.armeria.server.ServerBuilder;
+import com.linecorp.armeria.testing.junit4.server.ServerRule;
+
+import io.netty.handler.codec.http.HttpHeaderNames;
+import reactor.core.publisher.Flux;
+import reactor.test.StepVerifier;
+
+public class JsonLinesTest {
+    private static final ObjectMapper mapper = new ObjectMapper();
+
+    @ClassRule
+    public static ServerRule rule = new ServerRule() {
+        @Override
+        protected void configure(ServerBuilder sb) throws Exception {
+            sb.service("/seq/publisher",
+                    (ctx, req) -> JsonLines.fromPublisher(Flux.just("foo", "bar", "baz", "qux")))
+                    .service("/seq/stream",
+                            (ctx, req) -> JsonLines.fromStream(
+                                    Stream.of("foo", "bar", "baz", "qux"), MoreExecutors.directExecutor()))
+                    .service("/seq/custom-mapper",
+                            (ctx, req) -> JsonLines.fromPublisher(
+                                    Flux.just("foo", "bar", "baz", "qux"),
+                                    new ObjectMapper().enable(SerializationFeature.INDENT_OUTPUT)))
```

Comment:
Could you use a POJO that can be converted to multi-lines instead of `"foo"` which can't be multi-lines. 😆

Reply:
Done